src/us/kbase/narrativemethodstore/test/FullServerTest.java

```diff
@@ -1299,6 +1315,34 @@ public void testDynamicRepos() throws Exception {
 	    }
 	}
 
+	@Test
+	public void dynamicRepoWithValidFileTypes() throws Exception {
+		/* Tests that registering a repo with the valid_file_types key preserves that
+		 * key when fetching the module spec.
+		 */
+		final String moduleName = "nms_fake_type_test";
+		final String gitUrl = "https://github.com/kbasetest/nms_fake_type_test";
+		final String methodId = moduleName + "/run_" + moduleName;
+		/* Ideally we'd register the repo via the client, but auth is set up to use a remote
+		 * server, so that's not possible currently without using a pre-made token.
+		 * Long term set up a local auth server and run it as part of the tests like other
+		 * repos.
+		 */
+		// CLIENT.registerRepo(new RegisterRepoParams().withGitUrl(gitUrl));
+		SERVER.getLocalGitDB().registerRepo(admin1, gitUrl, null);
+		// note apps and methods are different. This apparently is a method.
+		final List<MethodSpec> spec = CLIENT.getMethodSpec(new GetMethodParams()
+				.withIds(Arrays.asList(methodId)).withTag("dev"));
+		System.out.println(spec);
```

Comment:
is this a leftover from debugging or should it be here?

Reply:
Cruft, removed.